Lib/difflib.py

```diff
@@ -1054,6 +1052,8 @@ def IS_LINE_JUNK(line, pat=re.compile(r"\s*(?:#\s*)?$").match):
     False
     """
 
+    if pat is None:
+        return line.strip() == '' or line.strip() == '#'
```

Comment:
```suggestion
        stripped = line.strip()
        return stripped == '' or stripped == '#'
```

Reply:
but now we have object creation, maybe we should leave `line.strip() in (...)`

Reply:
I'm not sure. just reacted